homeassistant/components/nibe_heatpump/__init__.py

```diff
@@ -0,0 +1,242 @@
+"""The Nibe Heat Pump integration."""
+from __future__ import annotations
+
+import asyncio
+from datetime import timedelta
+import logging
+
+from nibe.coil import Coil
+from nibe.connection import Connection
+from nibe.connection.nibegw import NibeGW
+from nibe.exceptions import CoilNotFoundException, CoilReadException
+from nibe.heatpump import HeatPump, Model
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_IP_ADDRESS, CONF_MODEL, Platform
+from homeassistant.core import CALLBACK_TYPE, HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import DeviceInfo, async_generate_entity_id
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import (
+    CONF_CONNECTION_TYPE,
+    CONF_CONNECTION_TYPE_NIBEGW,
+    CONF_LISTENING_PORT,
+    CONF_REMOTE_READ_PORT,
+    CONF_REMOTE_WRITE_PORT,
+    CONF_WORD_SWAP,
+    DOMAIN,
+)
+from .utils import TooManyTriesException, retry
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Instead of defining a logger in multiple places, why not define it once in `const.py`?

```python
LOGGER = logging.getLogger(__package__)
```

Reply:
Just not seen it done that way, but why not.